lib/public/views/LhcFills/Overview/LhcFillsOverviewModel.js

```diff
@@ -29,6 +30,8 @@ export class LhcFillsOverviewModel extends OverviewPageModel {
         super();
 
         this._stableBeamsOnly = stableBeamsOnly;
+
+        this._debouncedLoad = debounce(this.load.bind(this), 200);
```

Comment:
I think that the debounce is not useful, it might make sense when user fill in text but not the way we do it.

Reply:
Done